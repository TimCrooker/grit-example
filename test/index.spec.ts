import { getGenerator } from 'gritenv';
import path from 'path'

const generator = path.join(__dirname, '..')

describe('Name of the group', () => {
	it('defaults', async () => {
		const grit = await getGenerator({
			generator,
			mock: true,
			answers: {
				name: 'test',
				description: 'test',
				pluginTemplate: true,
				username: 'test',
				email: 'test',
				website: 'test',
			}
		})
		await grit.run()
	
		expect(await grit.getOutputFiles()).toMatchSnapshot()
	})
	

});


