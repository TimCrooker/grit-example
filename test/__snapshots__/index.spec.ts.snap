// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name of the group defaults 1`] = `
Array [
  ".gitignore",
  "LICENSE",
  "README.md",
  "package.json",
]
`;
